guides/vectorizing_guide.ipynb

```diff
@@ -0,0 +1,3643 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !pip install vectorhub[encoders-text-tfhub-windows] # If you are on windows
</pre><p>remove</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/RelevanceAI/RelevanceAI/pull/675/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
users need to install this to use the default vectorizers